src/params/index.ts

```diff
@@ -121,43 +209,60 @@ export async function getParamsByPath(argv: GetParamsByPathArgs): Promise<number
     Recursive: argv.recursive,
     WithDecryption: argv.decrypt
   };
-  const parameters: aws.SSM.ParameterList = await paginateAwsCall((args) => ssm.getParametersByPath(args), args, 'Parameters');
+  const parameters: aws.SSM.ParameterList = await paginateAwsCall((p) => ssm.getParametersByPath(p), args, 'Parameters');
```

Comment:
`p` is a bit misleading here as the argument is actually the same args that are passed in later in the call rather than a path.